### ISSUE TITLE
add-graphql-datasource fails on table using reserved keyword as name

### ISSUE THREAD

**Issue by vic-blt** [aws-amplify/amplify-cli#5525]
**Describe the bug**
`amplify api add-graphql-datasource` fails if the datasource contains a table using a reserved keywork as name (e.g. `order`).

Command output:
```
Using datasource: Aurora Serverless, provided by: awscloudformation
? Provide the region in which your cluster is located: eu-central-1
? Select the Aurora Serverless cluster that will be used as the data source for your API: <cluster>
✔ Fetched Aurora Serverless cluster.
? Select the database to use as the datasource: <database>
BadRequestException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'order' at line 1
    at Object.extractError (/home/<user>/.nvm/versions/node/v13.7.0/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/protocol/json.js:52:27)
    at Request.extractError (/home/<user>/.nvm/versions/node/v13.7.0/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/protocol/rest_json.js:55:8)
    at Request.callListeners (/home/<user>/.nvm/versions/node/v13.7.0/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/sequential_executor.js:106:20)
    at Request.emit (/home/<user>/.nvm/versions/node/v13.7.0/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/sequential_executor.js:78:10)
    at Request.emit (/home/<user>/.nvm/versions/node/v13.7.0/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/request.js:688:14)
    at Request.transition (/home/<user>/.nvm/versions/node/v13.7.0/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/request.js:22:10)
    at AcceptorStateMachine.runTo (/home/<user>/.nvm/versions/node/v13.7.0/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/state_machine.js:14:12)
    at /home/<user>/.nvm/versions/node/v13.7.0/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/state_machine.js:26:10
    at Request.<anonymous> (/home/<user>/.nvm/versions/node/v13.7.0/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/request.js:38:9)
    at Request.<anonymous> (/home/<user>/.nvm/versions/node/v13.7.0/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/request.js:690:12)
There was an error adding the datasource
```

**Amplify CLI Version**
4.29.5

**To Reproduce**
- Create an Aurora (MySQL 5.7) 2.07.1 Serverless DB Cluster
- Import a database with a table using a reserved keyword as name (e.g. `order`)
- Execute `amplify api add-graphql-datasource`

**Desktop:**
 - OS: Ubuntu 18.04.5
 - Node Version: v13.7.0

**Comment by vic-blt** [comment]
Same issue https://github.com/aws-amplify/amplify-cli/issues/2293

**Comment by ammarkarachi** [comment]
@vic-blt Thanks for the issue and the PR! we will review and get back to you soon as possible

**Comment by edwardfoyle** [comment]
The fix for this has been released in CLI v 4.32.0
